Microsoft/nni
# Code coverage report

To see python code coverage for integration test:
1. cd nni/test
2. run it.sh
3. check reports in coverhtml directory
